src/style.css

```diff
@@ -1 +1,105 @@
-/* add styles here */
+/* stylelint-disable font-family-name-quotes */
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
let's move this style block after @font-face;
so that at first we connected the fonts and then there was all the information about the styles